api/src/main/java/org/pmiops/workbench/db/model/WorkspaceUserRole.java

```diff
@@ -0,0 +1,62 @@
+package org.pmiops.workbench.db.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+
+@Entity
+@Table(name = "user_workspace")
+public class WorkspaceUserRole {
+  private long userWorkspaceId;
+  private User user;
+  private Workspace workspace;
+  private String role;
+
+
+
+  @Id
+  @GeneratedValue
+  @Column(name = "user_workspace_id")
```

Comment:
Arguably we could drop this column and just make (user ID, workspace ID) the primary key.